### PR TITLE
[MM-28328][MM-28329] Click "Getting Started" in lower LHS to go back to Next Steps View

### PR DESCRIPTION
**Pull request by nickmisasi**
#### Summary
When clicking away from the getting started or tips & next steps experiences the user couldn't return to it by clicking the Getting Started or Tips and Next Steps section in the lower LHS. You can now do this by clicking in that area.

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-28328
https://mattermost.atlassian.net/browse/MM-28329
#### Related Pull Requests
n/a
#### Screenshots
Mousing over once clicking away should turn to a cursor (I can't screenshot a cursor)
image

### CODE REVIEW

**Review by matthewbirtch — CHANGES_REQUESTED** [review]
@nickmisasi can we add a hover state (same background color change as a channel item) for when 'Getting Started' is not selected and you mouse over it?

File: package-lock.json [hunk 1]
```diff
@@ -10624,14 +10624,14 @@
...
           "version": "3.9.0",
-          "resolved": "https://registry.npmjs.org/file-type/-/file-type-3.9.0.tgz",
+          "resolved": "http://registry.npmjs.org/file-type/-/file-type-3.9.0.tgz",
           "integrity": "sha1-JXoHg4TR24CHvESdEH1SpSZyuek=",
           "dev": true,
           "optional": true
         },
         "get-stream": {
           "version": "2.3.1",
-          "resolved": "https://registry.npmjs.org/get-stream/-/get-stream-2.3.1.tgz",
```
Comment (devinbinnie) [inline package-lock.json:10634]: Nit: remove package-lock changes

**Review by devinbinnie — APPROVED** [review]
One non-blocking comment

**Review by matthewbirtch — APPROVED** [review]
looks great!

File: components/next_steps_view/next_steps_view.tsx [hunk 2]
```diff
@@ -208,6 +212,11 @@ export default class NextStepsView extends React.PureComponent<Props, State> {
     renderMainBody = () => {
         const renderedSteps = this.props.steps.map(this.renderStep);
 
+        // If all steps are complete, don't render this and skip to the tips screen
+        if (this.getIncompleteStep() == null) {
```
Comment (ali-farooq0) [inline components/next_steps_view/next_steps_view.tsx:216]:
```suggestion
        if (this.getIncompleteStep() === null) {
```

File: components/sidebar/sidebar_next_steps/sidebar_next_steps.scss [hunk 3]
```diff
@@ -5,6 +5,11 @@
     align-self: flex-end;
     position: relative;
     background-color: var(--sidebar-bg);
+    cursor: pointer;
+
+    &:hover:not(.active) {
+        background-color: rgba(255, 255, 255, 0.08) !important;
```
Comment (ali-farooq0) [inline components/sidebar/sidebar_next_steps/sidebar_next_steps.scss:11]: it's generally discouraged to use `!important` is there any other way to show the background? maybe a more specific selector?

**Review by ali-farooq0 — APPROVED** [review]
Nice @nickmisasi!

File: components/sidebar/sidebar_next_steps/sidebar_next_steps.scss (hunk 3 above)
Comment (marianunez) [reply components/sidebar/sidebar_next_steps/sidebar_next_steps.scss:11]: Also, is there a theme color we can use here?

**Review by stevemudie — APPROVED** [review]
LGTM

File: components/sidebar/sidebar_next_steps/sidebar_next_steps.scss (hunk 3 above)
Comment (nickmisasi) [reply components/sidebar/sidebar_next_steps/sidebar_next_steps.scss]: Turns out don't actually need the important. Think I just forgot to remove it. Added a variable also.

**Pull request merged**